custom/dhis2/tasks.py

```diff
@@ -288,64 +214,60 @@ def gen_children_only_ours(domain):
             yield CommCareCase.wrap(doc)
 
 
-@periodic_task(run_every=crontab(minute=4, hour=4))  # Run daily at 04h04
-def sync_child_entities():
+# TODO: Use case forwarding, or form forwarding of registration forms
+@periodic_task(run_every=timedelta(minutes=5))  # Run every 5 minutes to match forwarded forms
+def sync_cases():
```

Comment:
How are you going to query to get all the enabled domains? As far as I can
see there isn't any view or index that would give you that information.
Actually I guess you can query the toggle to see which domains are enabled
in the toggle.